src/System.Net.Sockets/src/System/Net/Sockets/TCPListener.cs

```diff
@@ -139,6 +142,16 @@ public void Start(int backlog)
                 return;
             }
 
+            if (_serverSocket == null)
+            {
+                _serverSocket = new Socket(_serverSocketEP.AddressFamily, SocketType.Stream, ProtocolType.Tcp);
+            }
+
+            if (_exclusiveAddressUse)
+            {
+                _serverSocket.ExclusiveAddressUse = true;
+            }
```

Comment:
Can this just be:
```C#
_serverSocket.ExclusiveAddressUse = _exclusiveAddressUse;
```
so that we don't need to worry about the default value of `_serverSocket.ExclusiveAddressUse`?

Reply:
This conditional assignment exists in the current version of Stop method and the tests show that on some platforms the default value is true. Thus, I decided to keep it this way to avoid breaking the implicit contract.